docs/typescript.md

```diff
@@ -2,9 +2,77 @@
 title: "Typescript"
 ---
 
-Emotion includes TypeScript definitions for `styled` components and has type inferences for both html elements and React components.
+Emotion includes TypeScript definitions for `create-emotion`, `emotion`, `create-emotion-styled`, and `react-emotion` packages. These definition also could infer types for css property (with object syntax), and HTML/SVG tag name, and props types.
```

Comment:
Is there a circumstance where it can't infer types for css properties?
If not, could you replace "These definition also could infer types for css property" with "These definitions also infer types for css properties"?

Also, could you replace "HTML/SVG tag name" with "HTML/SVG tag names"?

Reply:
Allright. I will change them.